config/sync/webform.webform.contact.yml

```diff
@@ -251,13 +251,13 @@ handlers:
       cc_options: {  }
       from_mail: _default
       from_options: {  }
-      from_name: _default
+      from_name: ''
       reply_to: ''
       return_path: ''
       sender_mail: ''
       sender_name: ''
       subject: '[webform_submission:values:subject:raw]'
-      body: '[webform_submission:values:message:value]'
+      body: '<p>Hej [webform_submission:values:name]&nbsp;</p><p>Vi bekræfter hermed at vi har modtaget følgende forespørgsel:&nbsp;</p><p>Modtaget: [webform_submission:created]</p><p>Forespørgsel:<br>[webform_submission:values:subject]</p><p>[webform_submission:values:message]</p><p>---------</p><p>Vi vil besvare dig hurtigst muligt på følgende email: [webform_submission:values:email]&nbsp;</p>'
```

Comment:
I think English should be the standard and Danish translations is the job of DDF to manage